client/web/src/components/externalServices/ExternalServicePage.tsx

```diff
@@ -246,6 +252,19 @@ export const ExternalServicePage: FC<Props> = props => {
                 {isErrorLike(isDeleting) && <ErrorAlert className="mt-2" error={isDeleting} />}
                 {externalServiceAvailabilityStatus}
                 <H2>Information</H2>
+                {noAuthorization && (
+                    <Alert className="mt-2" variant="warning">
+                        This code host connection does not have authorization configured. Any repositories added by this
+                        code host will be accessible by all users on the instance.
```

Comment:
```suggestion
                        code host will be accessible by all users on the instance, even if another code host connection with authorization syncs the same repository.
```
Maybe? Not sure. Is quite a mouthful.

Reply:
I think being verbose here is better 👍